lib/link_text.dart

```diff
@@ -295,64 +235,68 @@ TextSpan LinkTextSpans(
       var valid = true;
       if (scheme?.isNotEmpty ?? false) {
         // we have to validate the scheme
-        valid = ALL_SCHEMES.contains(scheme!.toLowerCase());
+        valid = allSchemes.contains(scheme!.toLowerCase());
       }
       if (valid && (tldUrl?.isNotEmpty ?? false)) {
         // we have to validate if the tld exists
-        valid = ALL_TLDS.contains(tldUrl!.toLowerCase());
-        link = 'https://' + link;
+        valid = allTlds.contains(tldUrl!.toLowerCase());
```

Comment:
for constant globals soru personally prefers screaming snake case....

Reply:
Well, Dart code style... I'd prefer it too...

Reply:
screw dart coding style, then x3